scripts/vale-review.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# This runs Vale on updated files, checks if the Vale alerts are on new/modified lines, and if so, builds curl request for GitHub review comment API
+
```

Comment:
Not strictly required tbh. but useful for anyone who wants to poke at this in future.

```suggestion
# Check if jq is installed
hash jq 2>/dev/null || { echo >&2 "Error: jq is not installed"; exit 1; }
```